src/components/Movie/index.tsx

```diff
@@ -11,20 +11,17 @@ import { MovieType } from "@/util/types";
  *  or a link to an external source. Each instance of this component is
  *  clickable and routes to /movies/:id
  */
-export default function Movie({ movieData }: { movieData: MovieType }) {
-  const link = `/movies/${movieData.id}`;
+export default function Movie({ movieData }: { movieData: MovieProps }) {
+  const link = `/movies/${movieData.title}`;
```

Comment:
Is this URL safe? Is it unique? What happens if two movies have the same name?

It's often a good idea to include something title-like in the URL (because it's easier for humans to understand and can be good for SEO), usually as a "slug" using something like [slugify](https://www.npmjs.com/package/slugify). But then always also include something that is guaranteed to be unique, e.g. an ID number or UUID string.

You could do something like:

```ts
  const link = `/movies/${movieData.id}-${slugify(movieData.title)}`;
```

And then only actually use the ID part, i.e. the part before the first `-` when you try to retrieve the movie on the target page (the page that this links to).